### PR TITLE
GG-34603 Split suites

### PR DESCRIPTION
**Pull request by Berkof**
[GG-34603](https://ggsystems.atlassian.net/browse/GG-34603)

### CODE REVIEW

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite.java [hunk 1]
```diff
@@ -181,6 +101,10 @@
 import org.apache.ignite.testframework.junits.DynamicSuite;
 import org.junit.runner.RunWith;
 
+import java.util.ArrayList;
```
Comment (ygerzhedovich) [inline modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite.java:104]: wrong order  for import java.* packages

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite10.java [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+import org.apache.ignite.internal.processors.cache.distributed.near.GridCacheNearTxExceptionSelfTest;
+import org.apache.ignite.internal.processors.cache.distributed.near.GridCachePartitionedStorePutSelfTest;
+import org.apache.ignite.internal.processors.cache.distributed.replicated.GridCacheReplicatedTxExceptionSelfTest;
+import org.apache.ignite.internal.processors.cache.expiry.IgniteCacheAtomicLocalExpiryPolicyTest;
+import org.apache.ignite.internal.processors.cache.local.GridCacheLocalTxExceptionSelfTest;
+import org.apache.ignite.testframework.GridTestUtils;
+import org.apache.ignite.testframework.junits.DynamicSuite;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
```
Comment (ygerzhedovich) [inline modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite10.java:104]: wrong order

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite11.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+import org.apache.ignite.internal.processors.cache.ConcurrentCacheStartTest;
+import org.apache.ignite.internal.processors.cache.IgniteCacheReadThroughEvictionsVariationsSuite;
+import org.apache.ignite.internal.processors.cache.PartitionsExchangeOnDiscoveryHistoryOverflowTest;
+import org.apache.ignite.internal.processors.cache.distributed.dht.NotMappedPartitionInTxTest;
+import org.apache.ignite.internal.processors.cache.distributed.dht.atomic.IgniteCacheAtomicProtocolTest;
+import org.apache.ignite.testframework.GridTestUtils;
+import org.apache.ignite.testframework.junits.DynamicSuite;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
```
Comment (ygerzhedovich) [inline modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite11.java:39]: wrong order

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite12.java [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+import org.apache.ignite.internal.processors.cache.transactions.TxCrossCacheMapOnInvalidTopologyTest;
+import org.apache.ignite.internal.processors.cache.transactions.TxCrossCacheRemoteMultiplePartitionReservationTest;
+import org.apache.ignite.internal.processors.cache.transactions.TxLocalTrackerWithoutTxRecordLoggingTest;
+import org.apache.ignite.internal.processors.cache.transactions.TxWithKeyContentionSelfTest;
+import org.apache.ignite.testframework.GridTestUtils;
+import org.apache.ignite.testframework.junits.DynamicSuite;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
+import java.util.Collection;
```
Comment (ygerzhedovich) [inline modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite12.java:50]: the same

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite13.java [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+import org.apache.ignite.internal.processors.cache.distributed.dht.topology.TombstoneClearingCountersTest;
+import org.apache.ignite.internal.processors.cache.distributed.dht.topology.TombstonesManagementTest;
+import org.apache.ignite.internal.processors.cache.transactions.TxRecoveryOnCoordniatorFailTest;
+import org.apache.ignite.internal.processors.cluster.ClusterNameBeforeActivation;
+import org.apache.ignite.testframework.GridTestUtils;
+import org.apache.ignite.testframework.junits.DynamicSuite;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
+import java.util.Collection;
```
Comment (ygerzhedovich) [inline modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite13.java:52]: order

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite5.java [hunk 6]
```diff
@@ -67,6 +48,10 @@
 import org.apache.ignite.testframework.junits.DynamicSuite;
 import org.junit.runner.RunWith;
 
+import java.util.ArrayList;
```
Comment (ygerzhedovich) [inline modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite5.java:51]: order

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite7.java [hunk 7]
```diff
@@ -16,61 +16,36 @@
...
-import org.apache.ignite.internal.processors.cache.transactions.TxLocalTrackerWithoutTxRecordLoggingTest;
 import org.apache.ignite.internal.processors.cache.transactions.TxRollbackAsyncWithPersistenceTest;
-import org.apache.ignite.internal.processors.cache.transactions.TxWithKeyContentionSelfTest;
 import org.apache.ignite.internal.processors.cache.transactions.TxWithSmallTimeoutAndContentionOneKeyTest;
 import org.apache.ignite.testframework.GridTestUtils;
 import org.apache.ignite.testframework.junits.DynamicSuite;
 import org.junit.runner.RunWith;
 
+import java.util.ArrayList;
+import java.util.Collection;
```
Comment (ygerzhedovich) [inline modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite7.java:46]: order

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite9.java [hunk 8]
```diff
@@ -72,12 +43,14 @@
 import org.apache.ignite.internal.processors.cache.transactions.TxPartitionCounterStatePutTest;
 import org.apache.ignite.internal.processors.cache.transactions.TxPartitionCounterStateTwoPrimaryTwoBackupsTest;
 import org.apache.ignite.internal.processors.cache.transactions.TxPartitionCounterStateWithFilterTest;
-import org.apache.ignite.internal.processors.cache.transactions.TxRecoveryOnCoordniatorFailTest;
-import org.apache.ignite.internal.processors.cluster.ClusterNameBeforeActivation;
 import org.apache.ignite.testframework.GridTestUtils;
 import org.apache.ignite.testframework.junits.DynamicSuite;
 import org.junit.runner.RunWith;
 
+import java.util.ArrayList;
```
Comment (ygerzhedovich) [inline modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite9.java:50]: order

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgnitePdsTestSuite.java [hunk 9]
```diff
@@ -77,6 +48,10 @@
 import org.apache.ignite.testframework.junits.DynamicSuite;
 import org.junit.runner.RunWith;
 
+import java.util.ArrayList;
+import java.util.Collection;
```
Comment (ygerzhedovich) [inline modules/core/src/test/java/org/apache/ignite/testsuites/IgnitePdsTestSuite.java:52]: order here and below

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite5.java [hunk 10]
```diff
@@ -119,36 +104,6 @@
...
-        GridTestUtils.addTestIfNeeded(suite, NotMappedPartitionInTxTest.class, ignoredTests);
-
-        GridTestUtils.addTestIfNeeded(suite, ConcurrentCacheStartTest.class, ignoredTests);
-
-        GridTestUtils.addTestIfNeeded(suite, IgniteCacheReadThroughEvictionsVariationsSuite.class, ignoredTests);
-
-        GridTestUtils.addTestIfNeeded(suite, ClientSlowDiscoveryTopologyChangeTest.class, ignoredTests);
-        GridTestUtils.addTestIfNeeded(suite, ClientSlowDiscoveryTransactionRemapTest.class, ignoredTests);
-
-        //GridTestUtils.addTestIfNeeded(suite, GridCacheAtomicPreloadSelfTest.class, ignoredTests);
```
Comment (ygerzhedovich) [inline modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite5.java:150]: I don'e see the line at any new suite

Comment (Berkof) [reply modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite5.java:150]: I remove commented code at all.

Comment (ygerzhedovich) [reply modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite5.java:150]: it not so, I see commented suites in new ones.

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite10.java (hunk 2 above)
Comment (Berkof) [reply modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite10.java]: Done

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite11.java (hunk 3 above)
Comment (Berkof) [reply modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite11.java]: Done

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite12.java (hunk 4 above)
Comment (Berkof) [reply modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite12.java]: Done

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite13.java (hunk 5 above)
Comment (Berkof) [reply modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite13.java]: Done

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite5.java (hunk 6 above)
Comment (Berkof) [reply modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite5.java]: Done

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite7.java (hunk 7 above)
Comment (Berkof) [reply modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite7.java]: Done

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite9.java (hunk 8 above)
Comment (Berkof) [reply modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite9.java]: Done

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgnitePdsTestSuite.java (hunk 9 above)
Comment (Berkof) [reply modules/core/src/test/java/org/apache/ignite/testsuites/IgnitePdsTestSuite.java]: Done

File: modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite5.java (hunk 10 above)
Comment (Berkof) [reply modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite5.java:150]: Сommented test in other suites removed

**Pull request merged**